mapnik_c_api.cpp

```diff
@@ -255,6 +269,15 @@ mapnik_image_blob_t * mapnik_image_to_png_blob(mapnik_image_t * i) {
     return blob;
 }
 
+const char * mapnik_version_string() {
+#if MAPNIK_VERSION >= 200100
+    return MAPNIK_VERSION_STRING;
+#else
+#define MAPNIK_C_API_STRINGIFY(n) #n
+    return "ABI " MAPNIK_C_API_STRINGIFY(MAPNIK_VERSION);
+#endif
```

Comment:
In the unlikely event of a real person ever using this with 2.0 or earlier nowadays, I wanted to remind them that the string "200000" did not mean mapnik 200000.0 but was a legacy encoding of "2.0.0". The ABI prefix seemed fitting. 
I am going to take the whole #if #else #endif out and just return MAPNIK_VERSION_STRING as soon as Travis is not dwelling in the 2.0 past any longer.